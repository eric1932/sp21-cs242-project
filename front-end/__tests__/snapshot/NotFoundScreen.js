import * as React from 'react';
import renderer from 'react-test-renderer';
import NotFoundScreen from "../../src/screens/NotFoundScreen";

const createTestProps = (props) => ({
  navigation: {
    navigate: jest.fn()
  },
  ...props
});

it('NotFoundScreen', () => {
  const tree = renderer.create(
    <NotFoundScreen {...createTestProps({})}/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
