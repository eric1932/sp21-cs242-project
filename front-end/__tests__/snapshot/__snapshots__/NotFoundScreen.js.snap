// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundScreen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
      "padding": 20,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "fontWeight": "bold",
      }
    }
  >
    This screen doesn't exist.
  </Text>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "marginTop": 15,
        "opacity": 1,
        "paddingVertical": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#2e78b7",
          "fontSize": 14,
        }
      }
    >
      Go to home screen!
    </Text>
  </View>
</View>
`;
